src/cmd/cli/command/commands.go

```diff
@@ -986,7 +986,9 @@ func configureLoader(cmd *cobra.Command) compose.Loader {
 		panic(err)
 	}
 
-	o.WorkingDir, err = f.GetString("cwd")
+	// the cli changes to the working directory at the start of the command so just set
+	// the working directory as the current directory
+	o.WorkingDir, err = os.Getwd()
 	if err != nil {
 		panic(err)
 	}
```

Comment:
So do we need this at all then?
```suggestion
```

Reply:
I think we can remove it. There's a corresponding change to the compose loader option too.